src/actions/index.ts

```diff
@@ -1,4 +1,6 @@
-export const addInput = input => {
+import { IInput } from '@autumn-platform/inputs';
+
+export const addInput = (input: IInput ) => {
```

Comment:
you don't need implement this action creator, it would be supplied by @autumn-ui/inputs-api-redux package